### DISCUSSION TITLE
Custom vnav and lnav

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#810] [Ideas]
Just checking this is in development?

**Comment by aeroniemi** [comment]
we have previously said that it isn't currently planned

**Reply by Simleybob** [reply]
I thought Omar has been working on that for like 9 months now, that’s what speed was saying anyway.

**Reply by fabian0hofmann** [reply]
![Bild_2022-04-18_200315345](https://user-images.githubusercontent.com/75206189/163852674-8d4e0860-f439-437d-8e51-d4b54a9b527a.png)
Not true.

**Reply by Simleybob** [reply]
![B92DA37B-DCEE-4D66-B03F-344BB1E4797D](https://user-images.githubusercontent.com/81443205/163855186-bd92d155-5eef-4593-974a-463943cdda7f.jpeg)

**Comment by Simleybob** [comment]
With VNAV not really working correctly, this is probably the most fundamental issue that people complain about, so you would get a lot of attention and a lot of thanks by making at least custom VNAV (if it doesn't take years) - for it to not have all the issues that default does. This is probably the most important item tbh, what people have been waiting for really.

**Reply by aeroniemi** [reply]
It is outside of the scope of the project, so unfortunately it is not likely to be added unless the scope changes.

**Reply by fabian0hofmann** [reply]
As shown above, it at least used to be in the scope of the project so when and why did that change?
